tailwind.config.js

```diff
@@ -5,4 +5,7 @@ module.exports = {
     extend: {},
   },
   plugins: [],
+  corePlugins: {
+    preflight: false,
+  },
```

Comment:
Why do we want to remove this? Tailwind's preflights seem useful for giving us consistent design.

Reply:
It does not work well with Antd's styles. Leaving this here if that's fine